spec/services/provider_interface/hesa_data_export_spec.rb

```diff
@@ -127,39 +113,24 @@
       let(:hesa_disabilities) { '55' }
 
       it 'exports this value' do
-        exported_data = CSV.parse(export_data, headers: true)
-        expect(exported_data.first['disabilities']).to eq('55')
+        expect(export_row['disabilities']).to eq('55')
       end
     end
 
     context 'when user does not have permission to view diversity information' do
       let(:provider_user) { create(:provider_user, providers: [training_provider]) }
 
       it 'shows diversity information as confidential' do
-        exported_data = CSV.parse(export_data, headers: true)
-        row = exported_data.first
-
-        expect(row['sex']).to eq('confidential')
-        expect(row['disabilities']).to eq('confidential')
-        expect(row['ethnicity']).to eq('confidential')
+        expect(export_row['sex']).to eq('confidential')
+        expect(export_row['disabilities']).to eq('confidential')
+        expect(export_row['ethnicity']).to eq('confidential')
       end
     end
 
     context 'when user is from the accredited provider' do
       let(:provider_user) { create(:provider_user, :with_view_diversity_information, providers: [accredited_provider]) }
 
-      it_behaves_like 'a full HESA export'
-    end
-
-    context 'when provider has courses in multiple recruitment cycles' do
```

Comment:
are we losing the courses in multiple recruitment cycle test here?

Reply:
:+1: I think I can still test this with multiple calls to `export_row`, will re-add.

Reply:
@cwrw Reinstated that test here https://github.com/DFE-Digital/apply-for-teacher-training/pull/5425/commits/aae73e56c96625d98459f84fc12f755adafed831